doc/doc-support/lib-function-docs.nix

```
@@ -1,12 +1,10 @@
-# Generates the documentation for library functons via nixdoc. To add
-# another library function file to this list, the include list in the
-# file `doc/functions/library.xml` must also be updated.
+# Generates the documentation for library functons via nixdoc.
 
-{ pkgs ? import ./.. {}, locationsXml }:
+{ pkgs ? import ./.. {}, locationsXml, libsets }:
```

```suggestion
{ pkgs, locationsXml, libsets }:
```

I don't think this file will be auto-called, making the default either duplicated or dead code.